definitions/npm/deepmerge_v4.x.x/flow_v0.100.x-/deepmerge_v4.x.x.js

```diff
@@ -0,0 +1,16 @@
+// Converted from https://github.com/TehShrike/deepmerge/blob/ad8cfddaa8f4b79f5e2c6943445a8b42377ec495/index.d.ts#L4
+
+declare module 'deepmerge' {
+    declare type Options = {
+        arrayMerge?: (target: any[], source: any[], options?: Options) => Array<any>,
+        clone?: boolean,
+        customMerge?: (key: string, options?: Options) => ?(x: any, y: any) => any,
+        isMergeableObject?: (value: Object) => boolean,
```

Comment:
`Object` is deprecated, if you can't gain a more precise type, then `{ ... }` should be used.